site-src/concepts/cluster-profile-api.md

```diff
@@ -0,0 +1,57 @@
+# ClusterProfile API Overview
+
+This document provides an overview of the [ClusterProfile API](https://github.com/kubernetes-sigs/cluster-inventory-api?tab=readme-ov-file#cluster-profile-api).
+
+![Alt](../images/cluster-profile-api.png "ClusterProfile API")
+
+A Cluster Profile is essentially an individual member of the Cluster Inventory that details the properties and status of a cluster.
+This API proposes a universal, standardized interface that defines how cluster information should be presented
+and interacted with across different platforms and implementations.
+
+You can read more details about the API in the [KEP-4322](https://github.com/kubernetes/enhancements/blob/master/keps/sig-multicluster/4322-cluster-inventory/README.md).
+
+## Terminology
+
+- **Cluster Inventory**: A conceptual term referring to a collection of clusters.
+
+- **Cluster Manager**: An entity that creates the ClusterProfile API object per member cluster,
+  and keeps their status up-to-date. Each cluster manager MUST be identified with a unique name.  
+  Each ClusterProfile resource SHOULD be managed by only one cluster manager. A cluster manager SHOULD
+  have sufficient permission to access the member cluster to fetch the information so it can update the status
+  of the ClusterProfile API resource.
+
+- **ClusterProfile API Consumer**: the person running the cluster managers
+  or the person developing extensions for cluster managers for the purpose of
+  workload distribution, operation management etc.
+

```

Comment:
Could you add one sentence to add the following key information: 1/ the relationship to namespace, 2/ the relationship to clusterSet.

something like:
```
Cluster Profiles are namespace-level resources and ClusterProfiles of the same namespace MUST be in the same clusterSet or no clusterSet at all.
```

Reply:
included them in the text.